src/infra/auth/jwt-passport.ts

```diff
@@ -95,11 +94,14 @@ export const createStrategy = (
 					const processedJwtUser = jwtUser as ResolvedUserPayload;
 					processedJwtUser.actor = user.actor.__id;
 					processedJwtUser.permissions = userPermissions;
-					return processedJwtUser;
+					done(null, processedJwtUser);
```

Comment:
Heads-up that there are more `return` statements higher.
How about wrapping the whole function in an iife so that we leave all the `return`s as is and we call `done` just once?

Reply:
Yeah I've changed the approach to callback-ifying this so it is much easier to get right now/hard to get wrong